HPXMLtoOpenStudio/resources/hpxml.rb

```diff
@@ -1551,6 +1550,12 @@ def is_interior
       return !is_exterior
     end
 
+    def is_adiabatic
+      if @exterior_adjacent_to == @interior_adjacent_to
+        return true
+      end
+    end
```

Comment:
These methods will either return true or nil, as opposed to true or false. Probably still works properly as written but I will address this.